src/catalog.h

```diff
@@ -707,7 +728,7 @@ typedef struct FormData_bgw_policy_drop_chunks
 {
 	int32 job_id;
 	int32 hypertable_id;
-	Interval older_than;
+	FormData_ts_interval older_than;
```

Comment:
@cevian Code later is much simpler when the struct is loaded here instead of having a datum pointer. It helps us encapsulate all of low level custom type reading/writing into a few places only. 

If we change this to datum, Any place that needs the `older_than` as a struct, has to do the loading itself.

Reply:
DISCUSS